physics/rigid_motion_test.cpp

```diff
@@ -188,6 +188,32 @@ TEST_F(RigidMotionTest, GroupoidInverse) {
   EXPECT_THAT(d2.velocity(), AlmostEquals(degrees_of_freedom_.velocity(), 6));
 }
 
+TEST_F(RigidMotionTest, SecondConstructor) {
+  DegreesOfFreedom<Selenocentric> const earth_dof_in_selenocentric =
+      geocentric_to_selenocentric_({Geocentric::origin,
+                                    Velocity<Geocentric>{}});
+  AngularVelocity<Selenocentric> const earth_rotation_in_selenocentric;
```

Comment:
`geocentric_to_selenocentric_` has vanishing angular velocity, so we are not testing this sign.

Involve terrestrial or lunar (or both: terrestrial_to_lunar, see above).